tests/conftest.py

```diff
@@ -151,8 +152,10 @@ def dc_inv(self, dc: NDArray[np.float64]) -> NDArray[np.float64]:
 
 
 @pytest.fixture(scope="session")
-def rng() -> np.random.Generator:
-    return np.random.default_rng(seed=42)
+def rng() -> np.random.Generator | Generator:
+    if array_lib != jax:  # how?
+        return np.random.default_rng(seed=42)
+    return Generator(seed=42)
```

Comment:
I have tried quite a few things to make this work, but I think the only thing that can be done here is [parameterizing the tests](https://docs.pytest.org/en/stable/example/parametrize.html#indirect-parametrization) that use this fixture.

I'll look at it more next week, but please let me know if you have any ideas in mind.